README.md

```diff
@@ -173,7 +173,7 @@ using (var stream = manager.GetStream())
 `GetReadOnlySequence` returns a [ReadOnlySequence<byte>](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.readonlysequence-1?view=netstandard-2.1) that can be used for zero-copy stream processing. For example, hashing the contents of a stream: 
 
 ```
-using (var stream = manager.GetStream())
+using (var stream = manager.GetStream() as RecyclableMemoryStream)
```

Comment:
The code on line 163 has the same issue.

Reply:
Thanks, good catch.